src/main/java/com/minecolonies/coremod/client/gui/WindowWareHouseBuilding.java

```diff
@@ -0,0 +1,36 @@
+package com.minecolonies.coremod.client.gui;
+
+import com.minecolonies.coremod.colony.buildings.BuildingHome;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.coremod.colony.buildings.BuildingHome'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)